README.md

```diff
@@ -1332,6 +1333,26 @@ See the [example-cron.yml](./.github/workflows/example-cron.yml) workflow.
 
 [![cron example](https://github.com/cypress-io/github-action/workflows/example-cron/badge.svg?branch=master)](.github/workflows/example-cron.yml)
 
+### Publish tests summary
+
+Sometimes users want to publish the test's summary to the pull request even if the current runtime environment does not support job summaries. 
+For example, a user running Cypress tests using a matrix and wanting to retrieve the tests summary for each matrix job.
+
+```yml
+name: info
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Cypress run
+        uses: cypress-io/github-action@v5
+        with:
+          publish-summary: true
```

Comment:
Change to
`publish-summary: false`

to follow the agreement to make 
`publish-summary: true`
the default selection.